src/depcache.erl

```diff
@@ -812,3 +840,6 @@ flush_message(Msg) ->
     after 0 ->
             ok
     end.
+
+callback(Type, Name, {M, F, A}) ->
+    spawn(M, F, [{Type, Name} | A]).
```

Comment:
Good point, @arjan.

I already replaced this with an `apply`, but if we need to wrap it in `try/catch` isn’t the `spawn` a simpler solution?